packages/ketchup/src/components/kup-data-table/kup-data-table.tsx

```diff
@@ -6083,9 +6083,11 @@ export class KupDataTable {
         };
 
         const addConfirmButton = () => {
-            this.#kupManager.keysBinding.register('enter', () =>
-                this.#handleUpdateClick()
-            );
+            this.#kupManager.keysBinding.register('enter', () => {
+                const bc: any = this.rootElement.shadowRoot.activeElement;
```

Comment:
any should be used only when there are no other alternatives

Reply:
I know, but I couldn't find a valid alternative. 
this.rootElement.shadowRoot.activeElement by default is an "Element", but Element don't have a blur function. However bc can't be an HTMLElement, because it lacks other functions.
I had no other ideas, but I'm open to suggestions in case